index.html

```diff
@@ -220,8 +220,8 @@ <h2>Scope</h2>
                         See the <a href="https://www.w3.org/2024/10/Manga_tpac_breakout.pdf">breakout presentation at TPAC 2024</a>.
                     </li>
                     <li>
-                        Introduce the usage of Web Assembly (<code>.wasm</code>) content in EPUB (e.g., as core media types),
-                        which may lead to changes on the usage of scripts in EPUB content in general.
+                        Explicitly define the relationship between the <code>&lt;script&gt;</code> element and exempt resources, for example in view of the inclusion of Web Assembly (<code>.wasm</code>) content in EPUB. See (postponed) 
```

Comment:
WASM files aren't referenced from `script`, so this could be misread. Maybe it would be helpful to add:

```suggestion
                        Explicitly define the relationship between the <code>&lt;script&gt;</code> element (including resources used by the referenced scripts) and exempt resources, for example in view of the inclusion of Web Assembly (<code>.wasm</code>) content in EPUB. See (postponed) 
```